content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+{{< warning >}}
+This guide relies heavily on the
```

Comment:
@rcaballeromx we'll need to add something like this at the top of the multicluster docs as well.